src/vip_client/classes/VipLauncher.py

```diff
@@ -1205,7 +1202,7 @@ def _get_exec_infos(cls, workflow_id: str) -> dict:
                 ),
             # Returned files (filtered information)
             "outputs": [] if not infos["returnedFiles"] else [
-                {"path": value} for value in infos["returnedFiles"]["output_file"] 
+                {"path": value} for value in infos["returnedFiles"].values()
```

Comment:
This has already been corrected, but not exactly in the same way
Can you please merge develop and call me to manage the conflict ?